digitalforms-api/src/test/java/ca/bc/gov/open/pssg/rsbc/digitalforms/controller/ApplicationFormControllerTests.java

```diff
@@ -42,59 +44,65 @@ public void init() {
 		MockitoAnnotations.initMocks(this);
 	}
 
+	// Need to update
 	@DisplayName("Get success - ApplicationFormController")
 	@Test
 	void getFormSuccess() {
```

Comment:
Just curious why the method name is getFormSuccess but we're checking for a NOT_FOUND in the response?

Reply:
I need to update the unit tests, I could comment it out instead.

Reply:
Do what works for you as we don't forget about it. Maybe leave a "TODO"

Reply:
Added a todo